plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -79,14 +88,6 @@ SecRule REQUEST_FILENAME "@endsWith /wp-comments-post.php" \
     t:none,\
     nolog,\
     ctl:ruleRemoveTargetById=931130;ARGS:url,\
```

Comment:
This line is now duplicated, right?

Reply:
I remove the ``931130`` exclusion and not the ``OWASP_CRS`` exclusion to stay safe in terms of FPs.